src/elements/kw-view-feed/kw-view-feed.html

```diff
@@ -174,6 +209,46 @@
                 '_routeChanged(route.path)',
                 '_queryMatchesChanged(detailQueryMatches, feedQueryMatches)'
             ],
+            _onRemixAction (e) {
+                let id = e.detail.id,
+                type = e.detail.type,
+                url = '';
+                switch (type) {
+                    case 'kano-draw':
+                        url = Kano.World.config.KANO_MAKEART_URL + (id ? '/share/' + id : '/');
+                        window.open(url);
+                        break;
+                    case 'make-apps':
+                        var pathname = window.location.pathname.replace('/shares/', '/remix/');
+                        url = Kano.World.config.KANO_CODE_URL + pathname;
+                        window.open(url);
+                        break;
+                    case 'make-adventures':
+                        url = Kano.World.config.KANO_ADVENTURES_URL + (id ? '/shared/' + id : '/');
+                        window.open(url);
+                        break;
+                    case 'make-pong':
+                        url = 'http://play-pong.kano.me' + (id ? '?project=' + id : '/')
+                        window.open(url);
+                        break;
+                    case 'kano-draw':
+                    case 'make-apps':
+                    case 'make-adventures':
+                    case 'make-pong':
+                        if (!Kano.Utils.isPi) {
+                            this.$.shopDialog.open();
+                        } else {
+                            this.$.loaderDialog.open();
+                            window.open("kano:share:" + id);
+                            setTimeout(() => {
+                                this.$.loaderDialog.close();
+                            }, 2500);
+                        }
+                        break;
+                    default:
+                        break;
+                }
+            },
```

Comment:
Sorry Joey, not sure I understand how these will open on the kit – except for the last `make-pong` case. Won't all these open on the web? I thought there would need to be an initial check to see if the app is embedded, and then the action should either open the web version or the kit version of the specific app. I totally might have misunderstood though – sorry if I'm just being dense!